# Don't assume we are in a git repo
# wpoely86@gmail.com
diff -ur bcalm-2.2.0.org/CMakeLists.txt bcalm-2.2.0/CMakeLists.txt
--- bcalm-2.2.0.org/CMakeLists.txt	2017-12-20 15:51:42.000000000 +0100
+++ bcalm-2.2.0/CMakeLists.txt	2018-08-31 15:07:53.080500643 +0200
@@ -19,6 +19,7 @@
 #############################
 # getting git version
 #from http://stackoverflow.com/questions/1435953/how-can-i-pass-git-sha1-to-compiler-as-definition-using-cmake
+IF(NOT DEFINED GIT_SHA1)
 exec_program(
     "git"
     ${CMAKE_CURRENT_SOURCE_DIR}
@@ -26,6 +27,7 @@
     OUTPUT_VARIABLE VERSION_SHA1 )
 
 add_definitions( -DGIT_SHA1="${VERSION_SHA1}" )
+ENDIF(NOT DEFINED GIT_SHA1)
 
 
 ################################################################################
